frontend/src/utils/firearms-utils/arme-alarme.ts

```diff
@@ -30,84 +30,97 @@ export const arme_alarme = {
   displayLabel: 'Arme d\'alarme',
   category: 'D',
   mention: 'Libre d\'acquisition et de détention',
-  options: {
-    arme_1: {
+  options: [
+    {
       label: '1. ZORAKI R2',
```

Comment:
No, why put a number in the labels ?? it's not coherent for a user to have number which do not follow, what use are they ?
![image](https://github.com/datalab-mi/Basegun/assets/24997639/dde4e18f-8fdd-47dc-98cf-6eafc8d9a919)